backend/graminsta/graminsta/settings.py

```diff
@@ -31,6 +31,7 @@
 # Application definition
 
 INSTALLED_APPS = [
+    'post',
```

Comment:
Add post to the end of the list to unify the code style.

Reply:
+1.

Normally, we would arrange the installed apps as

```
django apps

other third_party apps

our apps
```